imm_ptl_core/src/main/java/com/qouteall/immersive_portals/commands/PortalCommand.java

```diff
@@ -1312,6 +1324,34 @@ private static void invokeSetPortalDestination(
             }
         }
     }
+
+    private static void invokeSetPortalLocation(
+            CommandContext<ServerCommandSource> context,
+            Portal portal
+    ) throws CommandSyntaxException {
+        Vec3d pos = Vec3ArgumentType.getVec3(
+                context, "pos"
+                );
+        double posX = pos.getX();
+        double posY = pos.getY() + 1.000;
```

Comment:
Well I think that the command user may still be confused by its y level adding by 1. Maybe this adding by 1 can be removed.

Reply:
I was thinking that too as other commands like `set_portal_destination` do not do this so it will probably confuse them.